framework/PostProcessors/CrossValidation.py

```diff
@@ -215,7 +215,7 @@ def inputToInternal(self, currentInp, full = False):
       #here we do not make a copy since we assume that the dictionary is for just for the model usage and any changes are not impacting outside
       newInput = currentInput
 
-    if newInput.values().count(None) != 0:
+    if True in [x is None for x in newInput.values()]:
```

Comment:
any(x is None for x in newInput.values())

Reply:
Changed.